docs/libcurl/opts/CURLOPT_SSL_VERIFYSTATUS_FUNCTION.3

```diff
@@ -0,0 +1,86 @@
+.\" **************************************************************************
+.\" *                                  _   _ ____  _
+.\" *  Project                     ___| | | |  _ \| |
+.\" *                             / __| | | | |_) | |
+.\" *                            | (__| |_| |  _ <| |___
+.\" *                             \___|\___/|_| \_\_____|
+.\" *
+.\" * Copyright (C) 1998 - 2018, Daniel Stenberg, <daniel@haxx.se>, et al.
+.\" *
+.\" * This software is licensed as described in the file COPYING, which
+.\" * you should have received as part of this distribution. The terms
+.\" * are also available at https://curl.haxx.se/docs/copyright.html.
+.\" *
+.\" * You may opt to use, copy, modify, merge, publish, distribute and/or sell
+.\" * copies of the Software, and permit persons to whom the Software is
+.\" * furnished to do so, under the terms of the COPYING file.
+.\" *
+.\" * This software is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY
+.\" * KIND, either express or implied.
+.\" *
+.\" **************************************************************************
+.\"
+.TH CURLOPT_SSL_VERIFYSTATUS_FUNCTION 3 "11 Aprl 2019" "libcurl 7.64.2" "curl_easy_setopt options"
+.SH NAME
+CURLOPT_SSL_VERIFYSTATUS_FUNCTION \- User callback for certificate revocation checks.
+.SH SYNOPSIS
+#include <curl/curl.h>
+
+typedef CURLcode (*curl_ssl_verify_status_callback)(CURL *curl, void *ssl, void *userptr);
+										  
+CURLcode curl_easy_setopt(CURL *handle, CURLOPT_SSL_VERIFYSTATUS_FUNCTION, 
+	curl_ssl_verify_status_callback* user_verify_status_callback);
+.SH DESCRIPTION
+Pass a curl_ssl_verify_status_callback as parameter to use your custom callback for 
+certificate status verification.
+
+This option determines whether libcurl uses user callback to verify the status of the server cert.
+
+By default, curl checks leaf certificate using "Certificate Status Request" TLS extension 
+(aka. OCSP stapling). If the server does not support the TLS extension, the verification will fail.
+This callback can be used to customize certificate revocation checks. 
+For example, user can deside to check all certificates in certificate chain using 
+Certificate Revocation List (CRL). Or, user can deside to check all certificates in certificate chain 
+using Online Certificate Status Protocol (OCSP).  
+
+Note that if this option works only when CURLOPT_SSL_VERIFYSTATUS option is set to 1.
+
+.SH DEFAULT
+NULL
+.SH PROTOCOLS
+All TLS based protocols: HTTPS, FTPS, IMAPS, POP3S, SMTPS etc.
+.SH EXAMPLE
+See also cert_revocation_check.c in docs/examples directory for usage example.
+
+.nf
+static CURLcode user_verify_status_callback(CURL *curl, void *ssl, void *userptr)
+{
+	int* options = (int*)(userptr);
+
+	// do some ssl calls here
```

Comment:
I think we should avoid `//` comments since we don't allow them in curl code or examples.